src/shared/common/components/__tests__/workstack-tag.spec.jsx

```diff
@@ -18,6 +18,6 @@ describe('Workstack Tag component', () => {
                 tag: ['HS']
             }
         };
-        expect(Tags(props).length).toBe(1);
+        expect(Tags(props.row.tag)).toBeDefined();
```

Comment:
This is incorrect now? `row` is no longer passed in?

Reply:
I have updated the test cases now.